android/src/main/java/com/opentokreactnative/OTSessionManager.java

```diff
@@ -780,8 +780,14 @@ public void onStreamVideoDimensionsChanged(Session session, Stream stream, int w
             ConcurrentHashMap<String, Stream> mSubscriberStreams = sharedState.getSubscriberStreams();
             Stream mStream = mSubscriberStreams.get(stream.getStreamId());
             WritableMap oldVideoDimensions = Arguments.createMap();
-            oldVideoDimensions.putInt("height", mStream.getVideoHeight());
-            oldVideoDimensions.putInt("width", mStream.getVideoWidth());
+            if ( mStream != null ){
+                oldVideoDimensions.putInt("height", mStream.getVideoHeight());
+                oldVideoDimensions.putInt("width", mStream.getVideoWidth());
+            }
+            else {
+                oldVideoDimensions.putInt("height", -1);
```

Comment:
I don't think we should set the `height` and `width` to `-1` if the stream doesn't exist. Instead, we should leave the `oldVideoDimensions` as empty.